examples/mongo/chart.ts

```diff
@@ -0,0 +1,12 @@
+import { Construct } from "constructs";
+import { Mongo, TalisChart, TalisChartProps } from "../../lib";
+
+export class MongoChart extends TalisChart {
+  constructor(scope: Construct, props: TalisChartProps) {
+    super(scope, { app: "example-mongo-app", ...props });
+
+    new Mongo(this, "redis-example", {
+      release: "v1.0",
```

Comment:
I'd like to see a real mongo tag here: https://hub.docker.com/_/mongo?tab=tags
So that the example can actually be applied.

Reply:
ok
will do